### DISCUSSION TITLE
my-journey-to-japanese-n3-level

### DISCUSSION THREAD

**Discussion by giscus** [kutyel/kutyel-hakyll#49] [Announcements]
# my-journey-to-japanese-n3-level

My journey so far into learning the amazing Japanese language!

https://flaviocorpa.com/my-journey-to-japanese-n3-level.html

**Comment by NicolasPCouts** [comment]
Hi Flavio, that was a great post, thank you for taking the time to share your history with learning japanese, thanks for the tips (i'm not using and will not use any plugins to skip the vocab on wanikani haha).
You said english is not your first language, do you mind sharing what's your native language? I'm native brazilian portuguese speaker, i think you might be one too by your name.

About my journey learning Japanese, it all started around March 2023 when I bought a book out of curiosity on Amazon called "Japanese Picture Dictionary", it was on sale and I thought why not give it a try?

From there, I got more interested and bought another book called "Japanese From Zero" which was a great book teaching some really basic vocab and grammar structures, it also had a good YouTube series on YouTube where the author of the book teaches more about the grammar points with examples, very good.

From there I massively started with the SRS's, started with Core 2K/6K deck on Anki, started with Wanikani (level 23 now) and started with Bunpro.

On December 2023 I did the JLPT N5 just as a way to check my progress and passed, and on July 2024 I did the JLPT N4 and passed as well, my original plan was to try N3 in December 2024 but this proved to be an unrealistic goal since my listening skills still need to improve to get to N3 Level, and I wouldn't have time to complete the entire N3 Grammar Deck on BunPro and also learn all of the N3 Kanji, so I moved my deadline to July 2025.

I'm currently 53/219 points on the N3 Grammar Deck, and I'm also starting to read more, especially because i'm starting to understand more, so reading has started to become fun and not just pure frustration, I'm using a website called Satori Reader, it has a lot of good stories written in different levels of Japanese and with deep dive explanations about the grammar they use on each episode you read, very good for reading.
I do need to spend more time with listening immersion if I want to pass the N3 next year, so i should start focusing more on that soon.

Nice to meet you man, thanks for the time you took to write this post.

**Reply by kutyel** [reply]
Hey Nicolas! Thanks very much for your reply and for taking the time to read this! 🤩 (and for giving me the little push I needed to write the whole thing haha).

I'm natively a Spanish from Spain speaker, so naturally English was easier for me to learn than any asian language. 😉 

I think the lifting for next year of your N3 goal was a wise thing to do and also very reasonable, impressive how you managed to clinch N5 and N4 so quickly, keep it up! 🙌🏻 

Thanks very much for your recommendation on Satori Reader, I will definitely have a look at it as I'm also struggling to find time to read in Japanese, started some manga ago without furigana but I got discouraged because it was really difficult (shogi related manga 😰 ). I hope with Satori maybe I will be able to find something more digestible.

Thanks and nice to meet you! 👏🏻
